config/calibrate.py

```diff
@@ -21,7 +25,21 @@
 # This sets up the reference catalog for Gen3.
 config.connections.astromRefCat = "ps1_pv3_3pi_20170110"
 config.connections.photoRefCat = "ps1_pv3_3pi_20170110"
+
+# Photometric calibration: use color terms
+config.photoCal.applyColorTerms = True
 config.photoCal.photoCatName = "ps1_pv3_3pi_20170110"
+colors = config.photoCal.match.referenceSelection.colorLimits
+# The following two color limits are adopted from obs_subaru for the HSC SSP survey
+colors["g-r"] = ColorLimit(primary="g_flux", secondary="r_flux", minimum=0.0)
+colors["r-i"] = ColorLimit(primary="r_flux", secondary="i_flux", maximum=0.5)
+config.photoCal.match.referenceSelection.doMagLimit = True
+config.photoCal.match.referenceSelection.magLimit.fluxField = "i_flux"
+config.photoCal.match.referenceSelection.magLimit.maximum = 22.0
+config.photoCal.colorterms.load(os.path.join(obsConfigDir, 'colorterms.py'))
+
+# Number of bright stars to use. Sets the max number of patterns that can be tested.
+config.astrometry.matcher.numBrightStars = 150
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)